backend/src/configuration.rs

```diff
@@ -0,0 +1,51 @@
+use serde::Deserialize;
+use subxt::ext::sp_core::crypto::SecretStringError;
+use subxt::ext::sp_core::sr25519::Pair;
+use subxt::ext::sp_core::Pair as PairTrait;
+use subxt::tx::PairSigner;
+use subxt::utils::AccountId32;
+use subxt::OnlineClient;
+
+use crate::tx::KiltConfig;
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct Configuration {
+    // Seed for attester
+    pub attester_did_seed: String,
+    // Seed for payer
+    pub payer_seed: String,
+    // Seed for attestation key
+    attester_attestation_seed: String,
```

Comment:
why is this not public?

Reply:
I do not need the seeds for my server. I have also removed the pub from the other seeds in: [b3f6c61](https://github.com/KILTprotocol/attester/pull/1/commits/b3f6c612bc2e80cb7d7016d792602408cdc8b135)

Is there any reason to make them pub?